include/WebServer.hpp

```diff
@@ -8,6 +8,7 @@
 #define HTTP_LINEBREAK		"\r\n"
 #define SPACE				" "
 #define WHITESPACE			" \t"
+#define PREFIX_SHEBANG		"#!"
```

Comment:
I mean personally I love it but can we get something more descriptive name for this :D

Reply:
ahaha sure, I could not think of anything more descreptive, so I'll take ideas :D

Reply:
Is this script prefix or smt? You can push it in the next thing too if you're working on the same thing still after this PR!